internal/topo/node/event_window_trigger.go

```diff
@@ -151,7 +158,11 @@ func (o *WindowOperator) execEventWindow(ctx api.StreamContext, inputs []*xsql.T
 						o.triggerTime = inputs[0].Timestamp
 					}
 					if windowEndTs > 0 {
-						inputs = o.scan(inputs, windowEndTs, ctx)
+						if o.window.Delay > 0 && o.window.Type == ast.SLIDING_WINDOW {
```

Comment:
Is it better to calculate the window end in the getNextWindow() to unify the processing?